frame/support/src/storage/bounded_vec.rs

```diff
@@ -191,54 +205,76 @@ impl<T, S: Get<u32>> BoundedVec<T, S> {
 		S::get() as usize
 	}
 
-	/// Forces the insertion of `s` into `self` retaining all items with index at least `index`.
+	/// Returns true of this collection is full.
+	pub fn full(&self) -> bool {
```

Comment:
Small bikeshedding here:

```suggestion
	pub fn is_full(&self) -> bool {
```